ide-support/revsaveasstandalone.livecodescript

```diff
@@ -731,11 +731,39 @@ command revSaveAsMacStandalone pStack, pRestoreFileName, pFolder, @xStandalonePa
       __RestoreStackFromTemporaryLocation pStack, pRestoreFileName, tOriginalFileName
    end repeat
    
+   if the platform is "MacOS" then
+      local tAppBundle
+      set the itemDel to slash
+      put item 1 to -4 of tStandalonepath into tAppBundle
+      
```

Comment:
Make sure you set the itemDelimeter back to comma after it is used, otherwise it might cause hard to spot errors in the future if more sections are added after this one.